lib/wallet/integration/src/Test/Integration/Scenario/API/Shared/Transactions.hs

```diff
@@ -1020,3 +932,184 @@ spec = describe "SHARED_TRANSACTIONS" $ do
          RequireSomeOf m xs   -> RequireSomeOf m (map (changeRole role) xs)
          ActiveFromSlot s     -> ActiveFromSlot s
          ActiveUntilSlot s    -> ActiveUntilSlot s
+
+     singleOutputTxTwoParty ctx sharedWal1 sharedWal2 = do
+        -- check we see balance from two wallets
+        rSharedWal1 <- getSharedWallet ctx (ApiSharedWallet (Right sharedWal1))
+        rSharedWal2 <- getSharedWallet ctx (ApiSharedWallet (Right sharedWal2))
+
+        let balanceExp =
+                [ expectResponseCode HTTP.status200
+                , expectField (traverse . #balance . #available)
+                    (`shouldBe` Quantity faucetUtxoAmt)
+                ]
+
+        verify (fmap (view #wallet) <$> rSharedWal1) balanceExp
+        verify (fmap (view #wallet) <$> rSharedWal2) balanceExp
+
+        wb <- emptyWallet ctx
+        let amt = (minUTxOValue (_mainEra ctx) :: Natural)
```

Comment:
@paweljakubas 

By the way: you can now discover the right minimum UTxO value by:
1. Attempt to create a transaction with an output that has an ada value less than the minimum. (Specifying 1 lovelace should work.)
2. Read back the required minimum from the error object returned.

Hopefully, we should eventually be able to get rid of this `minUTxOValue` function!